tools/mmp/driver.cs

```diff
@@ -379,6 +379,11 @@ static void Main2 (string [] args)
 			if (!targetFramework.HasValue)
 				targetFramework = TargetFramework.Default;
 
+			if (App.EnableDebug && App.MarshalObjectiveCExceptions == MarshalObjectiveCExceptionMode.Default) {
```

Comment:
This is the right place to set the default value: https://github.com/xamarin/xamarin-macios/blob/cd9a151c984d814fb4f5ac83291620fc206d4ec7/tools/common/Application.cs#L362-L377

Reply:
:man_facepalming: